cv32e40x/tests/cfg/b_ext_all.yaml

```diff
@@ -6,4 +6,8 @@ ovpsim: >
 #    --showoverrides
 #    --trace --tracechange --traceshowicount --monitornets
 cflags: >
+riscv_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+gnu_march:   rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+corev_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+llvm_march:  rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
```

Comment:
I see that "default.yaml" has `+enable_zba_extension=1`, but I don't see that here (nor in b_ext_abs.yaml). Is this right?

Reply:
+enable_zbX_extension are directives for riscv-dv, the directed b-tests do not use riscv-dv and as such do not require these flags